framework/src/org/apache/cordova/CordovaActivity.java

```diff
@@ -175,9 +175,13 @@ protected void createViews() {
         setContentView(appView.getView());
 
         if (preferences.contains("BackgroundColor")) {
-            int backgroundColor = preferences.getInteger("BackgroundColor", Color.BLACK);
-            // Background of activity:
-            appView.getView().setBackgroundColor(backgroundColor);
+            try {
+                int backgroundColor = preferences.getInteger("BackgroundColor", Color.BLACK);
+                // Background of activity:
+                appView.getView().setBackgroundColor(backgroundColor);
+            }catch (NumberFormatException e){
+                System.out.println(e.getMessage());
```

Comment:
Changes has been done as you said.
All checks have passed.
Thank you

On Thu, Jul 28, 2016 at 1:23 PM, Akila Wicky <asak999@gmail.com> wrote:

> Ok i'll make the changes
>
> On Thu, Jul 28, 2016 at 12:41 PM, jcesarmobile <notifications@github.com>
> wrote:
>
>> In framework/src/org/apache/cordova/CordovaActivity.java
>> <https://github.com/apache/cordova-android/pull/316#discussion_r72573856>
>> :
>>
>> > @@ -175,9 +175,13 @@ protected void createViews() {
>> >          setContentView(appView.getView());
>> >
>> >          if (preferences.contains("BackgroundColor")) {
>> > -            int backgroundColor = preferences.getInteger("BackgroundColor", Color.BLACK);
>> > -            // Background of activity:
>> > -            appView.getView().setBackgroundColor(backgroundColor);
>> > +            try {
>> > +                int backgroundColor = preferences.getInteger("BackgroundColor", Color.BLACK);
>> > +                // Background of activity:
>> > +                appView.getView().setBackgroundColor(backgroundColor);
>> > +            }catch (NumberFormatException e){
>> > +                System.out.println(e.getMessage());
>>
>> Can you use e.printStackTrace(); instead of System.out?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/apache/cordova-android/pull/316/files/1be6ae190cc5804ae73a12044d184e72bc026f9e#r72573856>,
>> or mute the thread
>> <https://github.com/notifications/unsubscribe-auth/AKzUNOndsuxrQa3NLQc7zyXXlN1TZYWTks5qaFYugaJpZM4JLF9t>
>> .
>>
>
>
>
> --
>
>
> *Thanks and Regards,*Akila Wickramarachchi
>
> Undergraduate at University Of Peradeniya
> <https://www.linkedin.com/profile/view?id=AAMAABg_m_sBqt7aECwMPaTfYpf-olFytcSylC4&trk=hp-identity-headline>
> Email : asak999@gmail.com
> Linkedin : Akila Wickramarachchi
> <https://www.linkedin.com/profile/view?id=AAMAABg_m_sBqt7aECwMPaTfYpf-olFytcSylC4&trk=hp-identity-name>
> Blog : http://akilacollection.blogspot.com/
> Github : https://github.com/akilawickey
>



-- 


*Thanks and Regards,*Akila Wickramarachchi

Undergraduate at University Of Peradeniya
<https://www.linkedin.com/profile/view?id=AAMAABg_m_sBqt7aECwMPaTfYpf-olFytcSylC4&trk=hp-identity-headline>
Email : asak999@gmail.com
Linkedin : Akila Wickramarachchi
<https://www.linkedin.com/profile/view?id=AAMAABg_m_sBqt7aECwMPaTfYpf-olFytcSylC4&trk=hp-identity-name>
Blog : http://akilacollection.blogspot.com/
Github : https://github.com/akilawickey